src/Microsoft.Health.Fhir.Core/Features/Persistence/IFhirDataStore.cs

```diff
@@ -11,6 +11,8 @@ namespace Microsoft.Health.Fhir.Core.Features.Persistence
 {
     public interface IFhirDataStore
     {
+        Task<bool> MergeAsync(IReadOnlyList<ResourceWrapper> resources, CancellationToken cancellationToken);
```

Comment:
As discussed, might not be needed for initial check-in

Reply:
Removed